docker-compose.yml

```diff
@@ -33,13 +33,16 @@ services:
     image: api
     ports:
       - 8001:8000
+      - 5678:5678
+      - 8181:8181
     restart: unless-stopped
     volumes:
       - ./api/swotvis:/swotvis
     build:
       context: ./api/
       dockerfile: Dockerfile
     working_dir: /swotvis
+#    command: python -m debugpy --listen 0.0.0.0:5678 -m uvicorn --host 0.0.0.0 --port 8000 --proxy-headers main:app --reload
```

Comment:
I vote we keep this command live in the docker-compose. This project only uses docker compose for dev.

Reply:
Got it thanks! When you say "it won't work" you don't mean that the app wont run, do you?

Reply:
Maybe something like https://github.com/mfussenegger/nvim-dap/discussions/846 could work

Reply:
Sorry that was a vague response, but yes the application won't run.

Reply:
I'll test it again on my end, maybe the failure was an issue caused by something else.